pkg/sharding_engine/simple/sql_se.go

```diff
@@ -0,0 +1,75 @@
+package simple
+
+import (
+	"context"
+	"fmt"
+	pb "github.com/anujga/dstk/pkg/api/proto"
+	"github.com/jmoiron/sqlx"
+	"go.uber.org/zap"
+	"strings"
+	"time"
+)
+
+type sqlSe struct {
+	db *sqlx.DB
+}
+
+func (s *sqlSe) CreatePartition(ctx context.Context, request *pb.PartitionCreateRequest) (*pb.PartitionRpcBaseResponse, error) {
+	p := request.GetPartition()
+	p.ModifiedOn = time.Now().Unix()
+	q := `INSERT INTO partition 
+    		(id, modified_on, worker_id, start, "end", url, desired_state, current_state)
+    	 VALUES
+			(:id, :modified_on, :worker_id, :start, :end, :url, :desired_state, :current_state)
+	`
+	l := zap.L().With(zap.Any("part", p))
+	if result, err := s.db.NamedExec(q, *FromProto(p)); err == nil {
```

Comment:
still to standard if flow;
1. check for err in next line;
1. first check error and return
1. rest of code is not in else